Classes/Engine/HandlebarsEngine.php

```diff
@@ -167,7 +167,18 @@ protected function getOptions() : array
                 },
                 'lookup' => function ($labels, $key) {
                     return isset($labels[$key]) ? $labels[$key] : '';
-                }
+                },
+                'ifEqual' => function () {
+                    $arguments = func_get_args();
+                    $context = array_pop($arguments);
+                    return count(array_unique($arguments)) === 1 ? $context['fn']() : ($context['inverse'] ? $context['inverse']() : '');
+                },
+                'ifAny' => function () {
+                    $arguments = func_get_args();
+                    $context = array_pop($arguments);
+                    $notEmptyArguments = array_filter($arguments);
```

Comment:
Have a look at the documentation of `array_filter` (http://php.net/manual/en/function.array-filter.php)
> If no callback is supplied, all entries of array equal to FALSE (see converting to boolean) will be removed.
Is this what you want and need?

Reply:
As long as the count stays 0 it's fine for me